Makefile

```diff
@@ -70,6 +70,7 @@ $(SSH_KEY_PUB_PATH): $(SSH_KEY_PRIV_PATH)
 
 clean: destroy-libvirt
 	rm -rf $(INSTALLER_WORKDIR)
+	rm -rf registry-conifg.json
```

Comment:
make it phony instead?

Reply:
Actually, If I'll make it phony it will always update the file which will cause the invalidation of the INSTALLER_BIN.

Reply:
Right